models/Body.js

```diff
@@ -28,6 +28,8 @@ const Body = sequelize.define('body', {
         defaultValue: '',
         validate: {
             notEmpty: { msg: 'Code should be set.' },
+            isAlpha: { msg: 'Code should contains only letters.' },
```

Comment:
> should contains 

Infinitive after should.

Reply:
This is mandatory, therefore MUST be used instead of SHOULD.

Reply:
Should is also used in the rest of MyAEGEE, so that is something we can change in a different PR.
```suggestion
            isAlpha: { msg: 'Code should contain only letters.' },
```